python/pyspark/sql/connect/functions.py

```diff
@@ -2466,10 +2471,6 @@ def udf(
 udf.__doc__ = pysparkfuncs.udf.__doc__
 
 
-def pandas_udf(*args: Any, **kwargs: Any) -> None:
```

Comment:
and to address Takuya's comment, I guess it'd pretty easy from a cursory look. We can just throw an exception here:

```
def pandas_udf(f=None, returnType=None, functionType=None):
    from pyspark.sql.functions import pandas_udf
    if is_remote() and "PYSPARK_NO_NAMESPACE_SHARE" not in os.environ:
        return pandas_udf(f, returnType, functionType)
    else:
        # Should not reach here in production because `PYSPARK_NO_NAMESPACE_SHARE` is
        # for test only, and users should have invoked regular `pyspark.sql.functions.pandas_udf`
        # if Spark Connect is enabled.
        raise RuntimeError("blah blah")
```